src/match/models.py

```diff
@@ -0,0 +1,45 @@
+from django.contrib.auth.models import User
+from django.db import models
+from location.models import Location
+from match.validators import validate_int_list
+from match.validators import validate_times_list
+
+
+class Match(models.Model):
+    status = models.CharField(max_length=20)
+    user_1 = models.ForeignKey(
+        User,
+        default=None,
+        null=True,
+        on_delete=models.SET_NULL,
+        related_name="matches_1",
+    )
+    user_2 = models.ForeignKey(
+        User,
+        default=None,
+        null=True,
+        on_delete=models.SET_NULL,
+        related_name="matches_2",
+    )
+    proposer_id = models.IntegerField(default=None, null=True)
+    accepted_ids = models.CharField(
+        default=None,
+        null=True,
+        validators=[validate_int_list],
```

Comment:
i thought you got rid of `validate_int_list`

Reply:
oops yes i'll do that